docs/tools/issue-tracking.md

```diff
@@ -154,6 +154,14 @@ Categorization is useful for finding issues, and it also determines how issues a
 
 Use the Labels field to apply any relevant labels to your issue.
 
+#### Confidentiality {:#confidentiality}
+
+Issues can be marked Confidential, which is a middle ground between being public and high security. If you have a security issue then consider instead reporting it as described at [Reporting Vulnerabilities](/security/reporting/).
```

Comment:
```suggestion
Issues can be marked Confidential, this is a useful "middle ground" status for issues which perhaps shouldn't be public but which are **not** security issues. If you have a security issue please report it as described at [Reporting a Security Vulnerability](/security/reporting/).
```

This could be a little more explicit.